lib/gsl/oper.rb

```diff
@@ -40,4 +40,4 @@ def /(other)
 
 end
 
-[Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
+[RUBY_VERSION > "2.4" ? Integer : Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
```

Comment:
Constant RUBY_VERSION string has patch version number, example "2.4.0" and always been greater than "2.4"